### DISCUSSION TITLE
Clean up duplicate submissions from past in the production database

### DISCUSSION THREAD

**Discussion by sonalranjit** [bikespace/bikespace#200] [General]
![Screenshot 2024-06-05 at 10 29 44 AM](https://github.com/bikespace/bikespace/assets/12193813/83270251-83a1-44a5-a60f-fc7e09b4b810)
There are various duplicate submissions in the production database that are duplicates that could be cleaned up

**Comment by tallcoleman** [comment]
I'd be happy to help with this @sonalranjit - have a bunch of duplicates flagged already in a google sheet (posted it in the slack)
